src/Microsoft.NetCore.Analyzers/Core/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1062,4 +1062,13 @@
   <data name="DoNotCreateTasksWithoutPassingATaskSchedulerMessage" xml:space="preserve">
     <value>Do not create tasks without passing a TaskScheduler</value>
   </data>
+  <data name="UseDefaultDllImportSearchPathsAttribute" xml:space="preserve">
+    <value>Use DefaultDllImportSearchPaths attribute for P/Invokes</value>
```

Comment:
>Use DefaultDllImportSearchPaths attribute for P/Invokes [](start = 11, length = 55)

Trying to think of a title kinda makes me think we should have a separate rule for checking the value.  What do you think?

Reply:
If you thumbs up the comment, does that mean you're going to add another rule?  🙂

Reply:
NP.

Reply:
Is that a "yes let's have a separate diagnostic if the value has unsafe bits", or a "no let's use the same diagnostic"?  🙂